keps/sig-cloud-provider/0013-build-deploy-ccm.md

```diff
@@ -0,0 +1,652 @@
+---
+kep-number: 13
+title: Cloud Provider Controller Manager
```

Comment:
Does this title need to be changed?

Reply:
Yes. How about "Switching To Cloud Provider Repo And Builds"